BraintreeCore/src/main/java/com/braintreepayments/api/core/TLSCertificatePinning.kt

```diff
@@ -1,91 +1,718 @@
-package com.braintreepayments.api.core
+package com.braintreepayments.api
```

Comment:
package name has changed in v5 - please put back the `.core` suffix.

Reply:
Done